spec/rubocop/cop/lint/suppressed_exception_spec.rb

```diff
@@ -25,6 +25,29 @@
         end
       RUBY
     end
+
+    context 'when empty rescue for `def`' do
+      it 'registers an offense for empty rescue without comment' do
+        expect_offense(<<~RUBY)
+          def foo
+            do_something
+          rescue
+          ^^^^^^ Do not suppress exceptions.
+          end
+        RUBY
+      end
+
+      it 'registers an offense for empty rescue with comment' do
+        expect_offense(<<~RUBY)
+          def foo
+            do_something
+          rescue
+          ^^^^^^ Do not suppress exceptions.
+            # do nothing
+          end
+        RUBY
+      end
```

Comment:
```suggestion
      context 'without comments'
        it 'registers an offense' do
          expect_offense(<<~RUBY)
            def foo
              do_something
            rescue
            ^^^^^^ Do not suppress exceptions.
            end
          RUBY
        end
      end

      context 'with a comment' do
        it 'registers an offense' do
          expect_offense(<<~RUBY)
            def foo
              do_something
            rescue
            ^^^^^^ Do not suppress exceptions.
              # do nothing
            end
          RUBY
        end
      end
```

Reply:
https://rspec.rubystyle.guide/#context-descriptions

Reply:
https://rspec.rubystyle.guide/#example-descriptions here you can see that is a good practice to never end `it` with a conditional

Reply:
Same comments apply to the next block of tests 🤗

Reply:
Good suggestion. However, many existing tests often include a context in `it` in this repository. There is no problem in maintenance, so I leave it as it is. Thank you for your review, again 😃

Reply:
Ok. You're welcome 🤗